src/components/styles/WorldMap.css

```diff
@@ -1,4 +1,16 @@
+body {
+  background-color: #66a0b7;
+}
+
 .WorldMap {
   display: flex;
   flex-direction: column;
+  align-items: center;
+}
+
+.filterInput {
```

Comment:
Have you thought about using `BEM` for better naming your classes? That will help when the project gets bigger